source/common/http/conn_manager_utility.cc

```diff
@@ -387,7 +385,7 @@ void ConnectionManagerUtility::mutateResponseHeaders(HeaderMap& response_headers
     const bool no_body =
         (!response_headers.TransferEncoding() && !response_headers.ContentLength());
     if (no_body) {
-      response_headers.insertContentLength().value(uint64_t(0));
+      response_headers.setIntegerContentLength(uint64_t(0));
```

Comment:
WDYT of spelling this setContentLength() ? There would be no confusion with the string-ref or string_view versions which have specialized names. 

Actually you could also use `setForwardedClientCert(absl::string_view)`, and let the C++ type system figure out which one to pick.

Only the string-ref version would need to be named something different, which makes sense; there the caller is making an explicit promise that the ref will outlive the headers.

Reply:
My concern here would be automatic conversions. IMO it's better to be explicit unless we are sure the auto conversions won't happen. Will defer to better C++ experts on this one.